acceptance/analyzer_test.go

```diff
@@ -605,6 +601,35 @@ func testAnalyzer(t *testing.T, when spec.G, it spec.S) {
 					h.AssertPathExists(t, filepath.Join(copyDir, "layers", "some-buildpack-id", "some-layer.sha"))
 					h.AssertPathExists(t, filepath.Join(copyDir, "layers", "some-buildpack-id", "some-layer.toml"))
 				})
+
+				when("CNB_REGISTRY_AUTH is not provided", func() {
+					when("DOCKER_CONFIG is set", func() {
+						it("succeeds", func() {
+							h.DockerRunAndCopy(t,
+								containerName,
+								copyDir,
+								analyzeImage,
+								"/layers",
+								h.WithFlags(
+									"--env", "DOCKER_CONFIG=/registry-docker-config",
+									"--network", "host",
+								),
+								h.WithBash(
+									// ensure docker config directory is root owned and NOT world readable
+									fmt.Sprintf(
+										"chown -R root /registry-docker-config; chmod -R 700 /registry-docker-config; %s -cache-image %s %s",
+										analyzerPath,
+										cacheImage,
+										appImage,
+									),
+								),
+							)
+
+							h.AssertPathExists(t, filepath.Join(copyDir, "layers", "some-buildpack-id", "some-layer.sha"))
```

Comment:
Same comment as [here](https://github.com/buildpacks/lifecycle/pull/409/files#r480407637) but for the cache.